docker-compose.yml

```
@@ -0,0 +1,51 @@
+version: '2.0'
```

I think we could go up to version `2.4`  with this file.

```suggestion
version: '2.4'
```